ops/terraform/modules/resources/bfd_server_log_alarms/main.tf

```diff
@@ -1,7 +1,47 @@
 locals {
-  alarm_arn        = var.alarm_notification_arn == null ? [] : [var.alarm_notification_arn]
-  notification_arn = var.alarm_notification_arn == null ? [] : [var.alarm_notification_arn]
-  ok_arn           = var.ok_notification_arn == null ? [] : [var.ok_notification_arn]
+  victor_ops_sns         = "bfd-${var.env}-cloudwatch-alarms"
+  bfd_test_slack_sns     = "bfd-${var.env}-cloudwatch-alarms-slack-bfd-test"
+  bfd_warnings_slack_sns = "bfd-${var.env}-cloudwatch-alarms-slack-bfd-warnings"
+  default_ok_sns         = "bfd-${var.env}-cloudwatch-ok"
+  # Each established environment has a different destination for which alarm notifications should
+  # route to. The below map maps each particular SNS (destination) to a particular alarm severity.
+  topic_names_by_env = {
+    prod = {
+      alert  = local.victor_ops_sns
+      notify = local.bfd_warnings_slack_sns
+      ok     = local.default_ok_sns
+    }
+    prod-sbx = {
+      alert  = local.victor_ops_sns
+      notify = local.bfd_warnings_slack_sns
+      ok     = local.default_ok_sns
+    }
+    test = {
+      alert  = null # test is expected to not always be running, so alerts could be false positives
```

Comment:
Is this true? I can't recall a time test wasn't running (save for a bad deployment or something)

Reply:
This isn't necessarily true just yet, but there was some discussion recently about the _possibility_ of shutting `test` down after hours or something similar and I believe this particular log availability alarm was discussed as potentially causing some noise if that were ever to happen. So, this is more preemptive than anything as we could _technically_ turn the alarm on right now without there being any false positives since `test` is always running.